packages/keystone/lib/Keystone/index.js

```diff
@@ -44,6 +44,8 @@ module.exports = class Keystone {
     onConnect,
     cookieSecret = 'qwerty',
     sessionStore,
+    secureCookies = process.env.NODE_ENV === 'production', // Default to true in production
+    cookieMaxAge = process.env.NODE_ENV === 'production' ? 1000 * 60 * 60 * 24 : null,
```

Comment:
Why make `cookieMaxAge` different between environments? I'm happy if this one is defaulted to something like 30 days _always_, and folks can overwrite it for their app.

Reply:
I'm fine with that too. Sasa wanted one day and I thought it might be be annoying in dev but I'm happy to make it 30 days and we can change that for the app Sasa is working on?